test/Microsoft.ML.TestFramework/Attributes/NotCentOS7FactAttribute.cs

```diff
@@ -19,6 +19,12 @@ public NotCentOS7FactAttribute() : base("These tests are not CentOS7 compliant."
         }
         protected override bool IsEnvironmentSupported()
         {
+            // Featurizers.dll must exist
+            if(!Microsoft.ML.TestFrameworkCommon.Utility.NativeLibrary.NativeLibraryExists("Featurizers"))
```

Comment:
>Microsoft.ML.TestFrameworkCommon [](http://example.com/codeflow?start=16&length=32)

Is it needed to explicity adding the namespace?

Reply:
Yeah. Without it visual studio couldn't resolve it correctly. Ambiguity or something like that. Let me get you the exact error, but yeah it needed the full namespace to resolve correctly.

Reply:
I actually may have fixed it... not sure why it was giving me an error the first time. They should be removed now.